RealmModelGenerator/EntitiesViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  EntitieViewController.swift
+//  RealmModelGenerator
+//
+//  Created by Zhaolong Zhong on 3/21/16.
+//  Copyright © 2016 QuarkWorks. All rights reserved.
+//
+
+import Cocoa
+
+
+class EntitiesViewController: NSViewController, EntitiesViewDelegate, EntitiesViewDataSource {
+    static let TAG = NSStringFromClass(EntitiesViewController)
+    
+    @IBOutlet weak var entitiesView: EntitiesView!
+    
+    private var schema:Schema = Schema()
+    private var model: Model?
+    private var entities:[Entity] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+    
+    override func viewWillAppear() {
+        entitiesView.delegate = self
+        entitiesView.dataSource = self
+        
+        if let currentModel = schema.getCurrentModel() {
+            model = currentModel
+        } else {
+            model = schema.createModel()
+        }
+
+        entities = model!.entities
```

Comment:
most of this logic is handled in viewDidLoad() it is equal to onViewCreated in Android
onViewWillAppear is the same as onStart in android